app/templates/_package.json

```diff
@@ -4,39 +4,50 @@
     "node": ">=4"
   },
   "devDependencies": {
-    "autoprefixer": "^7.1.2",
-    "babel-core": "^6.4.0",
-    "babel-preset-es2015": "^6.3.13",
+    <% if (includeBabel || includeSass) { %>
+    "gulp-plumber": "^1.0.1",
+    <% } %>
+    <% if (includeBabel) { %>
+    "@babel/core": "^7.1.2",
+    "@babel/preset-env": "^7.1.0",
+    "gulp-babel": "^8.0.0",
+    <% } %>
+    <% if (includeSass) { %>
+    "gulp-filter": "^5.1.0",
+    "gulp-sass": "^4.0.2",
+    <% } %>
+    "gulp-postcss": "^8.0.0",
+    "autoprefixer": "^9.3.0",
     "babel-register": "^6.5.2",
     "browser-sync": "^2.2.1",
-    "cssnano": "^3.10.0",
-    "del": "^2.2.0",
-    "gulp": "^3.9.0",<% if (includeBabel) { %>
-    "gulp-babel": "^6.1.1",<% } %>
-    "gulp-cache": "^0.4.2",
-    "gulp-eslint": "^3.0.0",<% if (includeSass) { %>
-    "gulp-filter": "^4.0.0",<% } %>
-    "gulp-htmlmin": "^3.0.0",
+    "cssnano": "^4.1.7",
+    "del": "^3.0.0",
+    "gulp": "^3.9.0",
+    "gulp-cache": "^1.0.2",
+    "gulp-eslint": "^5.0.0",
+    "gulp-htmlmin": "^5.0.1",
     "gulp-if": "^2.0.2",
-    "gulp-imagemin": "^3.0.1",
-    "gulp-load-plugins": "^1.2.4",<% if (includeBabel || includeSass) { %>
-    "gulp-plumber": "^1.0.1",
-    "gulp-postcss": "^7.0.0",<% } if (includeSass) { %>
-    "gulp-sass": "^3.0.0",<% } %>
-    "gulp-size": "^2.1.0",
+    "gulp-imagemin": "^4.1.0",
+    "gulp-load-plugins": "^1.2.4",
+    "gulp-size": "^3.0.0",
     "gulp-sourcemaps": "^2.2.0",
-    "gulp-uglify": "^2.0.0",
+    "gulp-uglify": "^3.0.1",
     "gulp-useref": "^3.0.0",
     "main-bower-files": "^2.5.0",
-    "run-sequence": "^1.2.2",
+    "mocha": "^5.2.0",
```

Comment:
Why is Mocha being added? Depending on whether Mocha or Jasmine is chosen, those test runners will be installed by generator-mocha and generator-jasmine respectively.

Reply:
ok!